tests/unittests/history-model/tst_HistoryModelTests.cpp

```diff
@@ -271,6 +271,7 @@ private Q_SLOTS:
         delete model;
         model = new HistoryModel;
         model->setDatabasePath(fileName);
+        QTest::qWait(200);
```

Comment:
TRY_COMPARE should wait for the change by itself, so qWait should not be necessary... is it not waiting long enough?

Reply:
If i remember correctly there were timing related errors without the wait, but I don't find the build for it.
This was the original commit for it: https://github.com/ubports/morph-browser/commit/a0846f66671914314ee33bd4001d64d18c95260f#diff-561e4157ce1ef4f964f8183110ea6e4bb19b6fa0310e4c1831d931a7aec7cbae

Reply:
ok if it builds without it let's remove it for now. Maybe it just happened back then / with the old Qt version.